### PR TITLE
[Network] Fix no attribute issue for az network vhub connection create with --no-wait

### LINKED ISSUE #1176: az network vhub connection create with --no-wait  generates unexpected error
## Describe the bug

**Command Name**
`az network vhub connection create
Extension Name: virtual-wan. Version: 0.1.2.`

**Errors:**
```
'ClientRawResponse' object has no attribute 'virtual_network_connections'
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 206, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 608, in execute
    raise ex
cli/core/commands/__init__.py, ln 666, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
...
azure/cli/core/__init__.py, ln 490, in default_command_handler
    return op(**command_args)
cliextensions/virtual-wan/azext_vwan/custom.py, ln 170, in create_hub_vnet_connection
    return _get_property(poller.result().virtual_network_connections, connection_name)
AttributeError: 'ClientRawResponse' object has no attribute 'virtual_network_connections'
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az network vhub connection create --resource-group {} --vhub-name {} --name {} --remote-vnet {} --internet-security {} --no-wait`

## Environment Summary
```
Linux-5.3.0-7625-generic-x86_64-with-debian-buster-sid
Python 3.6.5
Shell: bash

azure-cli 2.0.78

Extensions:
virtual-wan 0.1.2

```

### PR DESCRIPTION
**Pull request by jsntcy**
This PR fix #1176.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S167

File: src/virtual-wan/azext_vwan/tests/latest/test_azure_vwan_vhub_commands.py [hunk 1]
```diff
@@ -22,3 +22,22 @@ def test_azure_vwan_vhub_basic_scenario(self, resource_group):
...
+            'vwan': 'clitestvwan',
+            'vhub': 'clitestvhub',
+            'connection': 'clitestvhubconnection',
+            'rg': resource_group
+        })
+
+        self.cmd('network vnet create -g {rg} -n {vnet}')
+        self.cmd('network vwan create -n {vwan} -g {rg} --type Standard')
+        self.cmd('network vhub create -g {rg} -n {vhub} --vwan {vwan}  --address-prefix 10.5.0.0/16 -l westus --sku Standard')
+        self.cmd('network vhub connection create --resource-group {rg} --vhub-name {vhub} --name {connection} --remote-vnet {vnet}', checks=[
```
Comment (MyronFanQiu) [inline src/virtual-wan/azext_vwan/tests/latest/test_azure_vwan_vhub_commands.py]: should we add `--no-wait` in the test?

Comment (jsntcy) [reply src/virtual-wan/azext_vwan/tests/latest/test_azure_vwan_vhub_commands.py]: Not too much value to test --no-wait as it returns None (Usually we don't test --no-wait)

---
In reply to: [398274212](https://github.com/Azure/azure-cli-extensions/pull/1450#discussion_r398274212) [](ancestors = 398274212)

**Comment by jsntcy** [comment]
will bump

---
In reply to: [381654314](https://github.com/Azure/azure-cli-extensions/pull/1450#issuecomment-381654314) [](ancestors = 381654314)

File: src/virtual-wan/azext_vwan/tests/latest/test_azure_vwan_vhub_commands.py (hunk 1 above)
Comment (MyronFanQiu) [reply src/virtual-wan/azext_vwan/tests/latest/test_azure_vwan_vhub_commands.py]: but this is a issue about no-wait?

**Pull request merged**